### DISCUSSION TITLE
SET_AUTO_OPERATE and TOGGLE_AUTO_OPERATE will disappear after saving and reloading.

### DISCUSSION THREAD

**Discussion by hxfmwj** [FakeFishGames/Barotrauma#14155] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

SET_AUTO_OPERATE and TOGGLE_AUTO_OPERATE will disappear after saving and reloading.
DOUBLE COILGUN does not have either of these interfaces.

![Q8R{Y5~05O8V@Y`{Z285%$4](https://github.com/FakeFishGames/Barotrauma/assets/166732182/0d4646bd-f8ec-4a86-8c0a-ceea58bed408)

![(W ()5`NF{$KVE$RAM`W7~V](https://github.com/FakeFishGames/Barotrauma/assets/166732182/1b24774f-91d4-4c04-bfe1-072127d29701)

### Reproduction steps

SET_AUTO_OPERATE and TOGGLE_AUTO_OPERATE will disappear after saving and reloading.
DOUBLE COILGUN does not have either of these interfaces.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.5.7.0 (Summer Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Jasontti** [comment]
Tested the fix, closing discussion as resolved.
